vertica/assets/monitors/vertica_replication_safety.json

```diff
@@ -22,8 +22,7 @@
 		"thresholds": {
 			"critical": 1,
 			"warning": 0,
-			"critical_recovery": 0,
```

Comment:
warning_recovery should have a different number.

Reply:
I think for these, would you mind deleting the critical recovery and warning if it's not too much hassle? I think the idea is that if it's `1` alert and if it's `<1` then recover. I think deleting the warning and critical recovery would essentially do this.

Reply:
will do